src/internal/SchemaBase.ts

```diff
@@ -175,6 +176,14 @@ export const Brand =
 export const Pattern: WithPattern2<URI>['pattern'] = (pattern, description) =>
   SC.make(S => S.pattern(pattern, description))
 
+/**
+ * @since 1.0.0
+ * @category Combinators
+ */
+export const InvMap: WithInvariant2<URI>['imap'] =
```

Comment:
Maybe this would be better at `Imap`, I can see it either way.  But I don't really like `IMap`

Reply:
Hmm. It would be a bit odd for this to be the one thing that isn't named the same as the internal methods. But the exposed API is essentially all Schemata so I'm not sure that really matters?

Reply:
We can rename it if it becomes a problem 👍